kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -57,7 +60,8 @@ class BatchJobSubmission(
     className: String,
     batchConf: Map[String, String],
     batchArgs: Seq[String],
-    recoveryMetadata: Option[Metadata])
+    recoveryMetadata: Option[Metadata],
+    isCleanupResourceFile: Boolean)
```

Comment:
rename to a variable to present the resource is a temporary uploaded one? 
1. cleanup it after batch terminated
2. do not check the resource for this case, see `KyuubiBatchSessionImpl::checkSessionAccessPathURIs`